tests/roles/ceph_backend_configuration/tasks/main.yaml

```diff
@@ -6,6 +6,8 @@
       CEPH_KEY=$($CEPH_SSH "cat /etc/ceph/ceph.client.openstack.keyring | base64 -w 0")
       CEPH_CONF=$($CEPH_SSH "cat /etc/ceph/ceph.conf | base64 -w 0")
 
+# TODO: (fpantano) We might need to update the Ceph keyring here if Manila is part of the
```

Comment:
@gouthampacha we still have to update this part as long as we discuss about [0]

[0] https://github.com/openstack-k8s-operators/data-plane-adoption/pull/189/files#r1386613613